pkg/test/frameworkSettings/netcoreapp3.1/settings.targets

```diff
@@ -3,6 +3,8 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <NETCoreAppMaximumVersion>3.1</NETCoreAppMaximumVersion>
     <RuntimeFrameworkVersion>$(MicrosoftNETCoreAppPackageVersion)</RuntimeFrameworkVersion>
+    <!-- This should be enabled once we have updating assembly versions in our targeting pack. -->
```

Comment:
To be clear, this should be enabled once we can ingest a netcoreapp3.1 ref-pack that has the change from this PR.